.env.example

```
@@ -380,7 +380,23 @@ ANTHROPIC_API_KEY=sk-ant-your-key-here
+# Get your API key from https://longcat.chat/platform/api_keys
+# Docs: https://longcat.chat/platform/docs/
+#
+#   CLAUDE_CODE_USE_OPENAI=1
+#   LONGCAT_API_KEY=your-longcat-key-here
+#   OPENAI_BASE_URL=https://api.longcat.chat/openai/v1
+#   OPENAI_MODEL=LongCat-2.0
+#
+# Or run: openclaude --provider longcat
+# Or configure interactively with /provider
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)